_includes/layer-cake.html

```diff
@@ -0,0 +1,145 @@
+<html>
```

Comment:
I don't think we need this tag here. We're already including this file in an html file.